Sources/Amplitude/AMPDefaultTrackingOptions.m

```diff
@@ -0,0 +1,67 @@
+//
+//  AMPDefaultTrackingOptions.m
+//  Copyright (c) 2023 Amplitude Inc. (https://amplitude.com/)
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "AMPDefaultTrackingOptions.h"
+
+@implementation AMPDefaultTrackingOptions
```

Comment:
Wonder if we should pick another name, we already have AMPTrackingOptions

Reply:
Think this is to be align with the TS SDK, for the interface name: `DefaultTrackingOptions`. In TS, it also has the `TrackingOptions`. Feel that the word `default` might not be the ideal word, but we are using it everywhere now. (cc @justin-fiedler @kevinpagtakhan )

Reply:
Ya that [matches Kotlin](https://github.com/amplitude/Amplitude-Kotlin/pull/126/files#diff-5b4d5af9d9a1252b33711a985b0638c257be1f498571509fa6b1259c75d20956R3). I think it's good to stay consistent.